blueprints/scaffold/index.js

```diff
@@ -24,38 +29,44 @@ module.exports = {
     var locals = buildNaming(options.entity.name);
     var resourcePath = locals.dasherizedModuleNamePlural;
 
-    var mirageConfig = this.insertIntoFile('app/mirage/config.js', [
-      'this.namespace = \'api\';',
-      'this.get(\'/' + resourcePath + '\');',
-      'this.get(\'/' + resourcePath + '/:id\');',
-      'this.post(\'/'+ resourcePath + '\');',
-      'this.del(\'/'+ resourcePath + '/:id\');',
-      'this.put(\'/'+ resourcePath + '/:id\');'
-    ].join('\n'), {
-      after: 'export default function() {\n'
-    });
-
     return RSVP.all([
-      mirageConfig,
       this.invoke('model', 'install', options),
-      this.invoke('scaffold-adapter', 'install', options),
       this.invoke('scaffold-template', 'install', options),
       this.invoke('scaffold-route', 'install', options),
       this.invoke('scaffold-mixin', 'install', options),
-      this.invoke('scaffold-acceptance-test', 'install', options)
+      this.invoke('scaffold-acceptance-test', 'install', options),
+      this.invoke('mirage-model', 'install', options),
+      this._addMirageEndpoints(locals, options)
     ]);
   },
   afterUninstall: function(options) {
     this._removeScaffoldRoutes(options);
     return RSVP.all([
       this.invoke('model', 'uninstall', options),
-      this.invoke('scaffold-adapter', 'uninstall', options),
       this.invoke('scaffold-template', 'uninstall', options),
       this.invoke('scaffold-route', 'uninstall', options),
       this.invoke('scaffold-mixin', 'uninstall', options),
       this.invoke('scaffold-acceptance-test', 'uninstall', options)
     ]);
   },
+  _addMirageEndpoints: function(locals, options) {
+    var mirageConfigPath = '/mirage/config.js';
+    fs.ensureFileSync(options.target + mirageConfigPath);
+
+    var templateLocals = {
+      resourcePath: locals.moduleNamePlural,
+    };
+
+    var templateStr = fs.readFileSync(__dirname + '/mirage-config.js.tpl', { encoding: 'utf8' });
+    var template = lodash.template(templateStr);
```

Comment:
I agree. I think that's one of the leftovers from my initial attempts. I think I'm going to redo this from scratch.